github3/pulls.py

```diff
@@ -254,6 +256,9 @@ def is_merged(self):
 
         :returns: bool
         """
+        if self.merged:
```

Comment:
Yep, I understand. I don't mind removing the short-circuiting if you feel that's a better solution, just thought it would work alright here since as far as I know a merged pull request can never change status.